tsconfig.json

```diff
@@ -8,5 +8,5 @@
     "resolveJsonModule": true,
     "jsx": "react-jsx",
     "allowSyntheticDefaultImports": true,
-  }
+  },
```

Comment:
This trailing comma can be problematic on a `.json` file. They're ok in Javascript or Typescript files, but we should avoid them on pure JSON files.

I believe your IDE automatically formatted this to include the line, but we should remove it.